src/tokens/index.ts

```diff
@@ -0,0 +1 @@
+export * from './tokens';
```

Comment:
I was unsure if this should export the individual token values as they are apart of the `tokens` export. @aaronccasanova happy to revert here but wanted some context.

Reply:
Yeah, I don't think it's necessary right now. I started that pattern for use cases where we might want to individually import tokens in components. For example: `motion` durations with `react-transition-group`
```tsx
import {motion} from '../tokens`
...
return (
  <CssTransition timeout={motion[...]} />
)
```

This update looks good to me 👍